The program being debugged has been started already.
Start it from the beginning? (y or n) Starting program: /home/baasdav/cis263/HashmapPrj/a.out 

Breakpoint 1, main (argc=1, argv=0x7fffffffe1b8) at src/main.cpp:41
41	    int index(hash1.hash1(hero.getName()));
42	    if(hash1.insert(hero, index))
my_hash<Superhero>::insert (this=0x7fffffffddd0, element=..., index=3) at ./include/my_hash.h:19
19	    data.at(index).at(0) = element;

Program received signal SIGABRT, Aborted.
0x00007ffff722d1f7 in raise () from /lib64/libc.so.6
Single stepping until exit from function raise,
which has no line number information.

Program terminated with signal SIGABRT, Aborted.
The program no longer exists.
The program is not being run.
The program is not being run.
Breakpoint 1 at 0x401743: file src/main.cpp, line 41.
Starting program: /home/baasdav/cis263/HashmapPrj/a.out 

Breakpoint 1, main (argc=1, argv=0x7fffffffe1b8) at src/main.cpp:41
41	    int index(hash1.hash1(hero.getName()));
Missing separate debuginfos, use: debuginfo-install glibc-2.17-196.el7.x86_64 libgcc-4.8.5-16.el7.x86_64 libstdc++-4.8.5-16.el7.x86_64
42	    if(hash1.insert(hero, index))
my_hash<Superhero>::insert (this=0x7fffffffddd0, element=..., index=3) at ./include/my_hash.h:19
19	    std::vector<T> item = data[index];
std::vector<std::vector<Superhero, std::allocator<Superhero> >, std::allocator<std::vector<Superhero, std::allocator<Superhero> > > >::operator[] (this=0x7fffffffddd0, __n=3)
    at /usr/include/c++/4.8.2/bits/stl_vector.h:771
771	      { return *(this->_M_impl._M_start + __n); }

Program received signal SIGSEGV, Segmentation fault.
0x0000000000402ce6 in std::vector<Superhero, std::allocator<Superhero> >::size (this=0x48) at /usr/include/c++/4.8.2/bits/stl_vector.h:646
646	      { return size_type(this->_M_impl._M_finish - this->_M_impl._M_start); }

Program terminated with signal SIGSEGV, Segmentation fault.
The program no longer exists.
